test/acceptance/integration/ISSUE-715 CPr-propagate X-Auth-Token header in forwarded requests/issue-715.feature

```diff
@@ -0,0 +1,157 @@
+      | providing_application     |
+      | /context_provider/service |
+    And build the standard context registration payload with the previous data and duration "P1M"
+    And a standard context registration is asked with the before information
+    # Query consult
+    And a new NGSI version "10" petition with the service "issue_715" and the subservice "/subservice"
+    And the following entities to consult
+      | entity_id | entity_type |
+      | Room1     | Room        |
+    And build the standard query context payload with the previous data
+    When a standard query context is asked with the before information
+    #Mock information
+    Then retrieve information from the mock
+    And headers of the last mock petition not contains the head "X-Auth-Token"
+    And the path in the last mock petition contains "service"
+    And there is "1" petitions requested to the mock
+    And clean the mongo database of the service "issue_715"
+
+  Scenario: Do not forward the X-Auth-Token for the context provider in a update operation
+    Given a started mock
+    And set the response of the context provider mock in path "/context_provider/service/updateContext" as "update_context_response_from_context_provider_xml"
+    # First registration
+    And a new NGSI version "9" petition with the service "issue_715" and the subservice "/subservice"
+    And the following entities to consult
+      | entity_id | entity_type |
+      | Room1     | Room        |
+    And the following attributes to consult
+      | attribute_name | attribute_type |
+      | att1           | att_type_1     |
+    And a context registrations with the before entities and attributes and the following providing applications
+      | providing_application     |
+      | /context_provider/service |
+    And build the standard context registration payload with the previous data and duration "P1M"
+    And a standard context registration is asked with the before information
+    # Update operation
+    And a new NGSI version "10" petition with the service "issue_715" and the subservice "/subservice"
+    And the following attributes to create
+      | attribute_name | attribute_type | attribute_value |
+      | att1           | att_type_1     | 25              |
+    And a context elements with the before attrs and the following entities
+      | entity_id | entity_type |
+      | Room1     | Room        |
+    And build the standard entity update payload with the previous data
+    When a standard context entity update is asked with the before information
+    #Mock information
+    Then retrieve information from the mock
+    And headers of the last mock petition not contains the head "X-Auth-Token"
+    And the path in the last mock petition contains "service"
+    And there is "1" petitions requested to the mock
+    And clean the mongo database of the service "issue_715"
+
+#Fixme: Repeat all tests with convenience operation, when developed
```

Comment:
Maybe we could add more precission to this, e.g.

"#Fixme: Repeat all tests with convenience operation, when developed. This is pending on the "big refactoring" in CPr funciontality being done in 1Q2015"